content/en/docs/install-operate/production-performance.md

```diff
@@ -0,0 +1,65 @@
+
+You can take actions to improve various aspects, notably reducing event drops. Some of these actions include increasing the kernel buffer size (although it comes with a higher memory cost) and tuning your Falco rules to minimize filtercheck matching operations while controlling the output volume. The key lies in being deliberate about what you monitor, taking into account your specific constraints. This approach may involve implementing enhanced monitoring in certain environments and adopting more targeted monitoring in challenging ones. For more in-depth insights, we recommend reading the [Adaptive Syscalls Selection in Falco](https://falco.org/blog/adaptive-syscalls-selection/) blog post. You can also experiment with the mentioned [configuration](https://github.com/falcosecurity/falco/blob/master/falco.yaml) options available to closely control Falco's performance and resource utilization. 
+
+Top metrics:
+
+- Kernel side and userspace event drops
+- Kernel tracepoint invocation counts to deduce overall server load
+- Userspace timeouts
+
+## Data Quality Assurance
+
+Furthermore, many of the [output fields](/docs/rules/style-guide/#output-fields) from the [supported output fields](/docs/reference/rules/supported-fields/) are derived from multiple events and mechanisms. As a result, they can never be perfect, and null values can occur. We are constantly aiming to improve the robustness in this regard. If you encounter such cases or have improvement ideas, we encourage you to [contribute](/docs/contribute/) to the project. 
+
+For example, the container info enrichment, while robust, depends on the speed of making API requests against the container runtime socket before the event is directed to the output channel.
+
+Let's consider another example, that is, the fields related to the process tree lineage (e.g. `proc.aname*`). Falco adds processes to a cache in userspace when a new process starts and removes them when the process exits. The goal is to maintain a current view of running processes on the Linux host at any given time. However, this also means that there are cases where the parent legitimately exits, re-parenting occurs, and/or PIDs get replaced or re-used. Therefore, a history of all `spawned_process` events is not equivalent to the current process tree on the system. Check out the Falco [rules](https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml) macro `container_entrypoint` for one such example. By the way, here is a resource to learn more about [process hierarchies](https://www.win.tue.nl/~aeb/linux/lk/lk-10.html). In summary, Falco aims to closely preserve the true system state, similar to the Linux kernel itself.
```

Comment:
@Andreagit97 kindly asking for some feedback as you are helping so much wrt improving the process tree, ty!

Reply:
IMO it's fine I would avoid going more deeper than this!  great work!

Reply:
No thank you for actually working on the process tree and making it more robust! Was wondering @Andreagit97 one of these days there should be a better doc around all these critical aspects, or a blog post if we don't want to add another official page ... let's check back later!